mne/event.py

```diff
@@ -885,3 +886,349 @@ def concatenate_events(events, first_samps, last_samps):
         events_out = np.concatenate((events_out, e2), axis=0)
 
     return events_out
+
+
+class ElektaAverager(object):
+    """Handles events and averaging categories defined in DACQ
+    (data acquisition) software of Elekta TRIUX/Vectorview systems.
+
+    Parameters
+    ----------
+    info : Info
+        An instance of Info where the DACQ parameters will be taken from.
+
+    Attributes
+    ----------
+    categories : list
+        List of averaging categories marked active in DACQ.
+    events : list
+        List of events that are in use (referenced by some averaging category).
+    reject : dict
+        Rejection criteria from DACQ that can be used with mne.Epochs.
+        Note that mne does not support all DACQ rejection criteria
+        (e.g. spike, slope)
+    flat : dict
+        Flatness criteria from DACQ that can be used with mne.Epochs.
+    """
+
+    # averager related DACQ variable names (without preceding 'ERF')
+    vars = ['magMax', 'magMin', 'magNoise', 'magSlope', 'magSpike', 'megMax',
+            'megMin', 'megNoise', 'megSlope', 'megSpike', 'eegMax', 'eegMin',
+            'eegNoise', 'eegSlope', 'eegSpike', 'eogMax', 'ecgMax', 'ncateg',
+            'nevent', 'stimSource', 'triggerMap', 'update', 'version',
+            'artefIgnore', 'averUpdate']
```

Comment:
are these elekta names? The syntax is really pythonic => `mag_max`, `mag_min`, `trigger_map` etc